packages/jest-snapshot/src/State.ts

```diff
@@ -19,30 +19,30 @@ import {
   testNameToKey,
   unescape,
 } from './utils';
-import {saveInlineSnapshots, type InlineSnapshot} from './inline_snapshots';
+import {saveInlineSnapshots, InlineSnapshot} from './inline_snapshots';
 
-export type SnapshotStateOptions = {|
-  updateSnapshot: SnapshotUpdateState,
-  getPrettier: () => null | any,
-  getBabelTraverse: () => Function,
-  expand?: boolean,
-|};
+export type SnapshotStateOptions = {
+  updateSnapshot: Config.SnapshotUpdateState;
+  getPrettier: () => null | any;
+  getBabelTraverse: () => Function;
+  expand?: boolean;
+};
 
-export type SnapshotMatchOptions = {|
-  testName: string,
-  received: any,
-  key?: string,
-  inlineSnapshot?: string,
-  error?: Error,
-|};
+export type SnapshotMatchOptions = {
+  testName: string;
+  received: any;
+  key?: string;
+  inlineSnapshot?: string;
+  error?: Error;
+};
 
 export default class SnapshotState {
```

Comment:
could you make all underscored properties `private`?

Reply:
How about removing underscore suffix also?

Reply:
I'd like them to keep the underscore so it's also visible for people not using TS. It also keeps the built diff down, fwiw

Reply:
you can also add `private` to the underscored functions